tests/test_flows.py

```diff
@@ -958,6 +958,34 @@ def test_topology(self):
         debug("Confirming topology query was received")
         self.check_log(env.agents[0], r"TOPOLOGY_QUERY_MESSAGE.*mid={:d}".format(mid))
 
+    def test_beacon_report_query(self):
+        # associated STA
+        sta = env.Station.create()
+
+        # for testing non existing STA, when want to test the error flow
+        # sta1 = env.Station.create()
+
+        debug("Connect dummy STA (" + sta.mac + ") to wlan0")
+        env.agents[0].radios[0].vaps[0].associate(sta)
+
+        # send beacon query request
+        # (please take a look at https://github.com/prplfoundation/prplMesh/issues/1272)
+        debug("Sending beacon report query to repeater:")
+        request = '{mac} '.format(mac=sta.mac)
+        request += '0x73 0xFF 0xFFFFFFFFFFFF 0x02 0x00 0x01 0x02 0x73 0x24 0x30 0x00'
+
+        debug(request)
+        mid = env.controller.dev_send_1905(env.agents[0].mac, 0x8011,
+                                           tlv(0x99, 0x0016, "{" + request + "}"))
+
+        self.check_log(env.agents[0],
+                       r"BEACON METRICS QUERY: "
+                       r"sending ACK message to the originator mid: {:d}".format(mid))
```

Comment:
Please don't add this kind of log checks any more. Instead, check for the ACK in the sniffer.